x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/non_text_fields_body.tsx

```diff
@@ -39,6 +40,10 @@ export const NonTextFieldsBody: React.FC = () => {
           </EuiTableRowCell>
           <EuiTableRowCellCheckbox>
             <EuiCheckbox
+              aria-label={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawArialLabel',
```

Comment:
Super minor spelling nit
```suggestion
                'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawAriaLabel',
```